### PR TITLE
Launch v1 of workflow plugin

### PR DESCRIPTION
**Pull request by SimeonGriggs**

### CODE REVIEW

File: src/tools/index.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
-import {Tool} from 'sanity'
 import {SplitVerticalIcon} from '@sanity/icons'
+import {Tool} from 'sanity'
```
Comment (runeb) [inline src/tools/index.ts:2]: Not sure if there is a guideline on this yet, but generally prefer a more explicit from of type imports like
```
import {type Tool} from 'sanity'
// or
import type {Tool} from 'sanity'
```

File: README.md [hunk 2]
```diff
@@ -1,9 +1,17 @@
+> This is a **Sanity Studio v3** plugin.
+
 # Sanity Workflow Demo Plugin Example
 
 With Sanity Studio you can [customize your content tools to support arbitrary workflows like assignment and content pipelines](https://www.sanity.io/docs/custom-workflows).
 
 This plugin is distributed as a **reference implementation** of these customization APIs and is not considered to be a feature-complete implementation of what workflow management requires in production. It is a starting point intended to be forked and customized to the needs of your organization and content creators.
```
Comment (runeb) [inline README.md:7]:
```suggestion
This plugin is distributed as an **example implementation** of customization APIs in the Sanity Studio V3 and is not considered to be a feature-complete implementation of what workflow management requires in production. It is meant as a starting point intended to be forked and customized to the needs of your organization and content creators, or simply as an illustration of what is possible in Sanity Studio V3.
```

Lets not call this a reference implementation. There are some patterns here I wish we could have avoided, like the WorkflowSignal component feeding into the context via top-level component rendering in the document form.

File: README.md [hunk 3]
```diff
@@ -1,9 +1,17 @@
...
 
+An intentional design choice of this plugin is that it **does not influence or modify whether a document is in draft or published**. It only tracks the values of a separate "metadata" document. In this implementation, an "Approved" document could be a draft but will still need publishing. "Approving" the document deletes the "metadata" and so removes it from the "Workflow" process. You choose if Publishing the document happens in the Studio like normal, using the [Scheduled Publishing plugin](https://www.sanity.io/plugins/scheduled-publishing) or the [Scheduling API](https://www.sanity.io/docs/scheduling-api#fa3bb95f83ed).
+
+This plugin is also considered feature complete and likely only to be supported for bug fixes and maintenance. Your feedback for workflow features you would like to see in Sanity Studio would be appreciated and can be [shared in our Slack community](https://slack.sanity.io/).
```
Comment (runeb) [inline README.md:11]:
```suggestion
This plugin is considered finished in its current form. Your feedback for workflow features you would like to see in Sanity Studio would be appreciated and can be [shared in our Slack community](https://slack.sanity.io/).
```

**Review by runeb — APPROVED** [review]
I have not had occasion to test this version, but looks good for its intended purpose!

File: src/tools/index.ts (hunk 1 above)
Comment (SimeonGriggs) [reply src/tools/index.ts:2]: There's an eslint rule that can enforce that. Remix ships with it.

File: README.md (hunk 2 above)
Comment (SimeonGriggs) [reply README.md:7]: Good call. I'd rather not have that either but there's presently no other way to feed a single async piece of data from a document up to a context that multiple badges and actions can read from.

**Pull request merged**